clang/lib/CodeGen/CGDebugInfo.cpp

```diff
@@ -3598,6 +3598,14 @@ llvm::DIMacroFile *CGDebugInfo::CreateTempMacroFile(llvm::DIMacroFile *Parent,
   return DBuilder.createTempMacroFile(Parent, Line, FName);
 }
 
+llvm::DILocation *CGDebugInfo::CreateSyntheticInline(llvm::DebugLoc Location,
+                                                     StringRef FuncName) {
+  llvm::DISubprogram *TrapSP =
```

Comment:
Nit. The name `TrapSP` doesn't make much sense here given this function isn't specifically for traps.

Reply:
Done.